gamecenter/public/views.py

```diff
@@ -5,12 +5,13 @@
 blueprint = Blueprint('public', __name__, static_folder="../static")
 
 
-@blueprint.route("/", methods=["GET"])
+@blueprint.route("/")
 def home():
-    # Handle logging in
+    """Return the home page"""
```

Comment:
I only put things like this in because my pep8 linter yells at me, and I'd rather it yell at me for more important things